client/my-sites/people/edit-team-member-form/edit-user-form.jsx

```diff
@@ -78,7 +86,7 @@ class EditUserForm extends React.Component {
 
 	getAllowedSettingsToChange() {
 		const { currentUser, user, isJetpack, hasWPCOMAccountLinked } = this.props;
-		const allowedSettings = [];
+		const allowedSettings = new Set();
 
 		if ( ! user.ID ) {
 			return allowedSettings;
```

Comment:
`allowedSettings` is no longer an array:
```suggestion
			return [];
```

Reply:
Good catch! 8da7790